### ISSUE TITLE
No Data labels showing for line chart in Highcharts.js and Narrator is not reading any info for Data points

### ISSUE THREAD

**Issue by Sheepu-dev** [highcharts/highcharts#14373]
#### Expected behavior
Data Labels should be visible for given series point and it should be announced by Screen Readers. 

#### Actual behaviour
I have two demo applications in which for one application all is working good but for the second application I am not getting Data labels for the chart and the screen reader is not reading any info. for Datapoints.

#### Live demo with steps to reproduce
Please refer to the Demo apps in which there is the only difference of Series Data points, please check with the Series Data point format difference for the demo apps and both the apps have different behavior with Screen readers(Narrator, NVDA). 

**Demo1:**
Not Working: http://jsfiddle.net/sheepu/Lpn8yuh1/8/
Chart Type: **line**

**Demo 2:**
Working:http://jsfiddle.net/sheepu/eut9xgzp/
Chart Type: **spline**

Please suggest.

#### Product version
HighCharts Version: v8.1.1
#### Affected browser(s)
Edge, Chrome, Mozilla

**Comment by PaulDalek** [comment]
Hi @Sheepu-dev

Thank you for reporting this issue. By the data labels you mean [series.labels](https://api.highcharts.com/highcharts/plotOptions.series.label)? If so, the reason why a label of the ‘Complete Recycles’ doesn’t show up is because it is positioned below the plot area. You can enable the `connectorAllowed` to make it appear.

**Demo**
http://jsfiddle.net/BlackLabel/Ltc6b4my/

**Comment by Sheepu-dev** [comment]
@PaulDalek Please refer to the attached screen below in Screen reader is not reading any info for the marked spikes in the Chart.

![image](https://user-images.githubusercontent.com/61267553/96545904-efbe9000-12c6-11eb-90ad-e9121de12afa.png)

**Comment by PaulDalek** [comment]
@Sheepu-dev 

Have you tried to set descriptions of individual data points through the [`accessibility.point`](https://api.highcharts.com/highcharts/accessibility.point)?

FYI @oysteinmoseng

**Comment by Sheepu-dev** [comment]
@PaulDalek Could you please help on how to write description for each data point in series. 
Please make changes in below app .
http://jsfiddle.net/sheepu/Lpn8yuh1/8/


I have already tried by giving description but it still not working for Any Screen Reader.

**Comment by oysteinmoseng** [comment]
This is a bug, and is happening because the series in question does not have markers, and Highcharts is not correctly compensating for it internally.

You can work around it by setting [`series.marker.enabled`](https://api.highcharts.com/highcharts/plotOptions.line.marker.enabled) either `true` or `false`: http://jsfiddle.net/oysteinmoseng/qso6u3tL/

Internal note: It looks like markers disabled due to `marker.enabledThreshold` are not picked up by the forced markers logic.

**Comment by oysteinmoseng** [comment]
Unstale, seems to still be valid
